packages/esify/lib/codemod.js

```diff
@@ -1,6 +1,5 @@
 var path = require('path');
```

Comment:
The `path` and `exec` module isn't used and is causing linting errors, causing tests to fail.